### PR TITLE
move package tutorial out of readme.

### LINKED ISSUE #46: Move tutorial out of readme
The tutorial in the readme should be moved out of the readme into it's own document.  This should be done as part of #41.

### PR DESCRIPTION
**Pull request by paynejacob**
closes #46

### CODE REVIEW

File: examples/registry/terraform/main/corral.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+variable "corral_name" {} // name of the corral being created
+variable "corral_user_id" {} // how the user is identified (usually github username)
+variable "corral_user_public_key" {} // the users public key
+variable "corral_public_key" {} // The corrals public key.  This should be installed on every node.
+variable "corral_private_key" {} // The corrals public key.  This should be installed on every node.
```
Comment (jakefhyde) [inline examples/registry/terraform/main/corral.tf:5]:
```suggestion
variable "corral_private_key" {} // The corrals private key.  This should be installed on every node.
```

**Review by jakefhyde — APPROVED** [review]
just a minor nit, otherwise LGTM!

File: docs/packages/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
+# How to Create a Package
+
+For this tutorial we will create a Corral package that sets up an OCI registry with authentication on a Digitalocean 
+droplet.
+
+# Anatomy of a Package
+
+To create a corral package we need to understand what goes into a package.  Packages consist of 3 components. The manifest
+defines what information is displayed to an end users and what the user is required to provide to use the package.  The
```
Comment (rmweir) [inline docs/packages/tutorial.md:9]:
```suggestion
defines what information is displayed to end users and what the user is required to provide to use the package.  The 
```
or
```suggestion
defines what information is displayed to an end user and what the user is required to provide to use the package.  The 
```

**Review by rmweir — CHANGES_REQUESTED** [review]
1 typo but otherwise tutorial looks good to me.

**Pull request merged**